### PR TITLE
Added meta-tags

### PR DESCRIPTION
**Pull request by carnarez**
@ritchie46 please review the spiel.

### CODE REVIEW

**Comment by ritchie46** [comment]
That's a promise. The javascrip lang😅

But there is someone working on a nodejs port, so it may happen. :)

**Comment by carnarez** [comment]
> That's a promise. The javascript lang. 😅
> But there is someone working on a nodejs port, so it may happen. :)

Might bring more contributors to the fold!

File: www/style.css [hunk 1]
```diff
@@ -164,14 +164,14 @@ nav .menu .pl-buttons > ul {
```
Comment (carnarez) [inline www/style.css:164]: The changes below are related to a clumsiness I fixed on the fly, nothing major.

**Comment by ritchie46** [comment]
LGTM :+1:

**Pull request merged**